### PR TITLE
Update Receive.md

### PR DESCRIPTION
**Pull request by BTCparadigm**
Trying to salvage something from #1088
thumb_my-pull-master-branch-my-pull-request-code-review-everytime-44869730

### CODE REVIEW

File: docs/using-wasabi/Receive.md [hunk 1]
```diff
@@ -61,15 +60,18 @@ However, an unspent transaction output, a bitcoin, is a chunk of money and is no
 This is just like a gold coin, with a unique amount of gold atoms, and either a pretty mintage or an old chipped broken coin.
 Because UTXOs are not fungible, it is very important to know exactly which coin is which, and that is done by carefully labeling.
 
+For example, if I create a new address to receive a `0.5 BTC` payment from Andrew for a laptop that I sold to him then the label has to be: `Andrew`.
+
+Wasabi does not label transactions, but the addresses that control the coins.
```
Comment (yahiheb) [inline docs/using-wasabi/Receive.md:65]: It does, when you get change back (cluster). I think this should just be removed.

File: docs/using-wasabi/Receive.md [hunk 2]
```diff
@@ -61,15 +60,18 @@ However, an unspent transaction output, a bitcoin, is a chunk of money and is no
...
+The label is not for describing the `reason for payment`, but rather to list the `involved parties`.
+Understanding this difference between labeling an address and describing a transaction is very important for our privacy.
```
Comment (yahiheb) [inline docs/using-wasabi/Receive.md:67]:
```suggestion
The label is not for describing the `reason for payment`, but rather to list those who know that this address is yours.
Understanding this difference between labeling an address and describing a transaction is very important for your privacy.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/using-wasabi/Receive.md (hunk 1 above)
Comment (BTCparadigm) [inline docs/using-wasabi/Receive.md:65]:
```suggestion
```

File: docs/using-wasabi/Receive.md [hunk 3]
```diff
@@ -14,16 +14,15 @@
 ## Generating addresses step-by-step
 
 1. Start Wasabi and open the wallet that you want to receive coins into.
-2. Go to the `Receive` tab.
-3. Specify the observers which are the entities who know about this address.
+2. Click the `Receive` button in the main view's top right corner.
+3. Type in the name of the entities who know about this address.
```
Comment (yahiheb) [inline docs/using-wasabi/Receive.md:18]:
```suggestion
3. Type in the name of the entities who know that this address is yours.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — APPROVED** [review]

**Pull request merged**